### DISCUSSION TITLE
Sensor clarification, inverter active vs input power

### DISCUSSION THREAD

**Discussion by Fettkeewl** [wlcrs/huawei_solar#237] [General]
IHi all, 

I need some help in regards to the image below, sensors created from this integration. 

**Can someone clarify if I've understood this correct:**
* **Active Power:** The actual usable power (DC -> AC)
* **Input Power:**  The actual created power from the panels (DC)

![image](https://user-images.githubusercontent.com/25577040/210746194-3f8ad4aa-4054-403e-a171-44cfa0a352da.png)

**Comment by Rhysgh** [comment]
Active power equals the total active output power of inverters. I take this as AC out.


Input power is DC from the PV.. although the documentation says this (Equals the total input power of all inverters.)

[SmartLogger ModBus Interface Definitions (2).pdf](https://github.com/wlcrs/huawei_solar/files/10351344/SmartLogger.ModBus.Interface.Definitions.2.pdf)

**Reply by Fettkeewl** [reply]
Thank you then the active is the one I want to log :)
And ty for the source

**Comment by scrichab** [comment]
when you have Luna connected to the first inverter then active_power_1 is als power from battery
when I need solar power (info for wallbox) then I use input_power_1 in my case + input_power_2
